src/main/java/smile/identity/core/ConfigHelpers.java

```diff
@@ -1,20 +1,18 @@
 package smile.identity.core;
 
-import java.util.Properties;
 
 public class ConfigHelpers {
-    private final static Properties properties = new Properties();
     private final static String TEST_ENV = "0";
     private final static String PROD_ENV = "1";
     private final static String TEST_SERVER = "https://testapi.smileidentity.com";
     private final static String PROD_SERVER = "https://api.smileidentity.com";
 
     public static String getVersion() {
-        return properties.getProperty("version");
+        return "2.0.0";
```

Comment:
Yea, I've temporarily set it as a string because it wasn't reading from the properties file correctly. I was planning to do a follow up PR to fix this, but if you would prefer I can resolve that in this pr @beastawakens